odm_sdk/settings/genestack_config.py

```diff
@@ -5,9 +5,9 @@
 
 import sys
```

Comment:
filename odm_sdk_config?

Reply:
Answer in the very next conversation.

Reply:
https://genestack.atlassian.net/browse/ODM-11243